plugin-catalog/src/components/plugins/InstalledList.tsx

```diff
@@ -81,8 +144,20 @@ export function PluginInstalledList() {
       }
     }
 
+    async function fetchOtherInstalledPlugins() {
```

Comment:
this function doesn't use `await` so `async` is not needed

Reply:
done!